cfme/common/vm.py

```diff
@@ -321,12 +320,11 @@ def find_quadicon(
         if use_search:
             try:
                 if not search.has_quick_search_box():
-                    # We don't use provider-specific page (vm_templates_provider_branch) here
-                    # as those don't list archived/orphaned VMs
+                    # TODO rework search for archived/orphaned VMs
                     if self.is_vm:
-                        navigate_to(self.provider, 'Instances')
+                        navigate_to(self, 'AllForProvider', use_resetter=False)
```

Comment:
same is here , additionally VM View in ems_infra controller doesn't provide Search box, imho - better to use this navigation